core/src/main/java/org/apache/calcite/sql/dialect/StarRocksSqlDialect.java

```diff
@@ -19,14 +19,7 @@
 import org.apache.calcite.avatica.util.TimeUnitRange;
 import org.apache.calcite.config.NullCollation;
 import org.apache.calcite.rel.type.RelDataType;
-import org.apache.calcite.sql.SqlAbstractDateTimeLiteral;
-import org.apache.calcite.sql.SqlAlienSystemTypeNameSpec;
-import org.apache.calcite.sql.SqlCall;
-import org.apache.calcite.sql.SqlDataTypeSpec;
-import org.apache.calcite.sql.SqlDialect;
-import org.apache.calcite.sql.SqlLiteral;
-import org.apache.calcite.sql.SqlNode;
-import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.*;
```

Comment:
@fanluoo calcite does not allow import.*

Reply:
> @fanluoo calcite does not allow import.*
Thank you for the reminder, Let me make some adjustments